src/fetchServices.ts

```diff
@@ -167,10 +167,11 @@ export async function exportApisToSheets() {
   const values = await getResults();
   values.unshift(['Service', 'Title', 'Group', 'HasSurface', 'InScope', 'ToS']);
```

Comment:
won't this lead to the first column being added multiple times, since we're no longer clearing A1?

Reply:
I think it will override